multisrc/src/main/java/eu/kanade/tachiyomi/multisrc/heancms/HeanCms.kt

```diff
@@ -182,14 +181,13 @@ abstract class HeanCms(
     }
 
     private fun createSearchResult(slug: String, title: String): SManga {
-        val thumbnailUrl = seriesSlugMap?.get(slug)?.thumbnailUrl ?: ""
-        val status = seriesSlugMap?.get(title)?.status ?: ""
+        val theSeries = seriesSlugMap?.get(slug)
 
         return SManga.create().apply {
-            url = "/series/$slug"
+            url = "/series/${theSeries?.slug}"
```

Comment:
Just in case the title is not found (and can happen sometimes since their All series pages doesn't include some titles for some unknown reason), it's better to include the old slug instead of letting it empty.

```suggestion
            url = "/series/${theSeries?.slug ?: slug}"
```

Rest seems good now, just this minor one that needs to be fixed 👍🏻.

Reply:
Actually need to always be the `slug` since it doesn't include the timestamp that changes, otherwise it will make some titles not showing the "In library" badge if the users already have it on their libraries.

The slug in the map is used to create the requests only, but it should not be stored in the Tachiyomi database.

Reply:
Just fix that one mistake I suggested and it will be good to merge, sorry about it, but that's the last one now 👍🏻.

Reply:
So It should be
` url = "/series/$slug"` ?